ddapm_test_agent/agent.py

```diff
@@ -451,6 +500,27 @@ async def handle_test_traces(self, request: Request) -> web.Response:
             traces = list((await self.traces()).values())
         return web.json_response(data=traces)
 
+    async def handle_test_apmtelemetry(self, request: Request) -> web.Response:
+        """Return requested telemetry events as JSON.
+
+        Telemetry events can be requested by providing a header X-Datadog-Runtime-Ids or
+        a query param runtime_ids.
+        """
+        raw_runtime_ids = request.url.query.get(
+            "runtime_ids", request.headers.get("X-Datadog-Runtime-Ids", "")
+        )
+        if raw_runtime_ids:
+            runtime_ids = raw_runtime_ids.split(",")
+            events: List[TelemetryEvent] = []
+            for rid in runtime_ids:
+                try:
+                    events.extend(await self._apmtelemetry_by_runtime_id(rid))
+                except KeyError:
+                    events.append({})
```

Comment:
append an empty event? Wonder if we should fail instead?

Don't feel super strongly about one way or the other.

Reply:
I was just trying to follow the pattern used for traces... I think?

I borrowed this idea, it is weird

Reply:
ah, yeah that is weird 🤷‍♂️